website/components/remote-plugin-docs/utils/resolve-nav-data.js

```diff
@@ -155,7 +161,7 @@ async function resolvePluginEntryDocs(pluginConfigEntry, currentPath) {
     const { nav_title, sidebar_title } = frontmatter
     const title = nav_title || sidebar_title || basename
     // construct sourceUrl (used for "Edit this page" link)
-    const sourceUrl = `https://github.com/${repo}/blob/${version}/${filePath}`
+    const sourceUrl = `https://github.com/${repo}/blob/${sourceBranch}/${filePath}`
```

Comment:
We now construct the `sourceUrl` from the `sourceBranch` rather than `version`. This `sourceUrl` is ultimately used to render the `"Edit this page"` link at the bottom of each plugin docs page.